tools/build_file_list.py

```diff
@@ -10,8 +10,8 @@
 parser.add_argument('dataset', type=str, choices=['ucf101', 'hmdb51', 'activitynet_1.2', 'activitynet_1.3'])
 parser.add_argument('frame_path', type=str, help="root directory holding the frames")
 parser.add_argument('--rgb_prefix', type=str, help="prefix of RGB frames", default='img_')
-parser.add_argument('--flow_x_prefix', type=str, help="prefix of x direction flow images", default='flow_x')
-parser.add_argument('--flow_y_prefix', type=str, help="prefix of y direction flow images", default='flow_y')
+parser.add_argument('--flow_x_prefix', type=str, help="prefix of x direction flow images", default='flow_x_')
```

Comment:
I think it doesn't matter to change this. Would you please remove these changes so I can merge your fix?